src/mongo/util/version_constants_gen.py

```diff
@@ -245,14 +245,32 @@ def generate_config_header(
     modules = ["enterprise"] if "build_enterprise_enabled" in extra_definitions_dict else []
     module_list = ",\n".join(['"{0}"_sd'.format(x) for x in modules])
 
-    psmdb_pro_features = []
-    if "fipsmode_enabled" in extra_definitions_dict:
-        psmdb_pro_features.append("FIPSMode")
-    if "fcbis_enabled" in extra_definitions_dict:
-        psmdb_pro_features.append("FCBIS")
-    if "oidc_enabled" in extra_definitions_dict:
-        psmdb_pro_features.append("OIDC")
-    psmdb_pro_feature_list = ",\n".join(['"{0}"_sd'.format(x) for x in psmdb_pro_features])
+    # For now, we just enable all the Percona-specific features by choosing
```

Comment:
IMHO this might be a little bit confusing. There is still an option to decide if some of the features should be enabled or disabled in the build (e.g. `oidc_enabled` switch). In such case the build would assert here. If we want to always build all of the features, can't we get rid of these `*_enabled/disabled` switches?